### ISSUE TITLE
Cannot create preexisting table when running tests for multiple jobs in Jenkins

### ISSUE THREAD

**Issue by paul-uz** [shelfio/jest-dynamodb#153]
We use Jenkins to test, build and deploy our code for multiple clients.

The jobs run in parallel, so when we write tests which use this library, we encounter an issue where only one job will pass the tests, with the rest failing with "Cannot create preexisting table", which makes sense, as they all use the same codebase, which all reference the same local address for jest-dynamodb, https://localhost:8000.

Is there anyway around this?

Would using random ports work, or does it have to be 8000?

**Comment by vladgolubev** [comment]
Yes, you can have multiple configs at different ports if you want to parallelize

<img width="950" alt="CleanShot 2022-07-19 at 22 57 41@2x" src="https://user-images.githubusercontent.com/3817380/179847050-5b501ea8-3694-4938-a7ad-7f072a532c92.png">

Alternatively, PRs are welcome to change the logic to optionally skip table creation if it exist already

**Comment by paul-uz** [comment]
@vladgolubev how do i setup the multiple configs?

How do I then make sure my code connects to the correct port?

Eg if i run two jobs, client A and client B, they use the same codebase, but I'd need to somehow tell A to use port 8000 and B to use 8001
